common/src/stack/ansible/plugins/inventory/stacki.py

```diff
@@ -0,0 +1,130 @@
+# @copyright@
+# Copyright (c) 2006 - 2020 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+    inventory: stacki
+    short_description: Returns hosts, groups, and vars from Stacki
+    description:
+        - Reads hosts info and attributes from Stacki.
+        - Returns host groups based on standard Stacki host selectors.
+        - Hosts have attributes attached as vars.
+"""
+
+EXAMPLES = """
+# Directly target hosts
+ansible backend-0-0 -m ping
+
+# Using standard host selectors
+ansible a_backend -m ping
+
+# In Ansible playbooks, you can use the selectors in the `hosts` attribute:
+- name: Ping my backends
+  hosts: a_backend
+  tasks:
+    - ping:
+
+# You can also use the --limit flag to `ansible-playbook`
+ansible-playbook --limit b_default playbook.yml
+
+# The following types of groups are created, with the UPPERCASE replaced
+# with the Stacki primitive:
+- a_APPLIANCE
```

Comment:
Consider documenting what the selector becomes on the stacki side, I.E. `a_APPLIANCE -> a:APPLIANCE`.

Reply:
I'll document it in more depth on the wiki, when it makes it into a release. The example section is already probably too long for the internal Ansible module docs.